apps/abstraxion-dashboard/components/AbstraxionWallets/index.tsx

```diff
@@ -23,19 +30,36 @@ export const AbstraxionWallets = () => {
     setAbstraxionError,
   } = useContext(AbstraxionContext) as AbstraxionContextProps;
 
+  const [errorMessage, setErrorMessage] = useState("");
+
   const { user } = useStytchUser();
   const stytchClient = useStytch();
   const session_jwt = stytchClient.session.getTokens()?.session_jwt;
   const session_token = stytchClient.session.getTokens()?.session_token;
 
-  const { aud, sub } = decodeJwt(session_jwt || "");
+  const keplr = getKeplr();
+  const { data: grazAccount } = useAccount();
+  const { client } = useAbstraxionSigningClient();
 
+  const { suggestAndConnect } = useSuggestChainAndConnect({
+    onSuccess: async () => await addKeplrAuthenticator(),
+  });
   const { disconnect } = useDisconnect();
-  const { data: account } = useAbstraxionAccount();
+
+  const { aud, sub } = session_jwt
+    ? decodeJwt(session_jwt)
+    : { aud: undefined, sub: undefined };
+
+  // TODO: More robust
```

Comment:
what is the robustness issue here?

Reply:
I might've just forgotten to delete this comment